extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -61,4 +61,21 @@ public static function onArticleSaveComplete(
 
 		return true;
 	}
+
+	/**
+	 * Add community page entry point to article page right rail module
+	 *
+	 * @param array $railModuleList
+	 * @return bool
+	 */
+	public static function onGetRailModuleList( Array &$railModuleList ) {
+		$context = RequestContext::getMain();
+
+		if ( ( $context->getUser()->isLoggedIn() && $context->getTitle()->inNamespace( NS_MAIN ) ) ||
+			   $context->getTitle()->isSpecial( 'WikiActivity' ) ) {
```

Comment:
We want the logged in check to also occur on WikiActivity, don't we?

Also, our conventions would be to format the multi-line if like this (assuming logged in should be checked regardless):
```php
$title = $context->getTitle();
if (
	$context->getUser()->isLoggedIn() &&
	( $title->inNamespace( NS_MAIN ) || $title->isSpecial( 'WikiActivity' ) )
) {
	// if block
}
```